docs/content/operate/5-faq.md

```diff
@@ -17,6 +17,10 @@
 - Note that setting up cosmovisor is strongly advised to support smooth network upgrades. See our [halovisor build scripts](https://github.com/omni-network/omni/tree/main/scripts/halovisor) for inspiration.
 - Note that before starting geth, it must first be initialised with the Omni Omega [`execution-genesis.json`](https://github.com/omni-network/omni/tree/main/lib/netconf/omega) file via `geth init`.
 
+### How to check if the validator is ready?
```

Comment:
Maybe explain what ready means: ready means the node is healthy and operating as expected, this includes consensus and execution chains being fully synced, having execution and consensus P2P peers, and producing new blocks. It will be unhealthy while syncing or P2P peers dropped to 0 of if consensus stalled.